templates/careers/_careers-progression-slides.html

```diff
@@ -2,7 +2,7 @@
 	<div class="p-careers-progression-carousel__slides">
 		<div class="p-careers-progression-carousel__slide" aria-hidden="false">
 			{{
-				image (
+				image(
```

Comment:
this should have looked different if you reformatted it after removing the space.